clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -1024,7 +1026,14 @@ inline sycl::event matmul(descriptor_ptr handle, matmul_desc_ptr compute_desc,
 
   if (compute_desc->_epilogue != epilogue_t::nop) {
     ::dnnl::post_ops matmul_ops;
-    matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_relu, 0.f, 0.f);
+    if (compute_desc->_epilogue == epilogue_t::relu) {
+      matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_relu, 0.f, 0.f);
+    } else if (compute_desc->_epilogue == epilogue_t::bias) {
+      matmul_ops.append_binary(::dnnl::algorithm::binary_add, bias_md);
+    } else if (compute_desc-> epilogue == epilogue::gelu_aux_bias) {
+      matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_gelu_erf, 0.f, 0.f);
```

Comment:
1. `CUBLASLT_EPILOGUE_GELU_AUX_BIAS` needs to output the `bias` result to `CUBLASLT_MATMUL_DESC_EPILOGUE_AUX_POINTER`. But here, your implementation does not.
2. Please write an E2E test to validate your implementation. I tried something similar before but encountered a runtime error or an incorrect result.

Reply:
Ok. Please help to review.